source/components/JsxParser.js

```diff
@@ -133,11 +133,20 @@ export default class JsxParser extends Component {
       .map(tag => tag.trim().toLowerCase()).filter(Boolean)
 
     if (/^(html|head|body)$/i.test(name)) return childNodes.map(c => this.parseElement(c))
-    if (blacklistedTags.indexOf(name.trim().toLowerCase()) !== -1) return undefined
+    const tagName = name.trim().toLowerCase()
+    if (blacklistedTags.indexOf(tagName) !== -1) {
+      onError(new Error(`The tag <${name}> is blocked, and will not be rendered.`))
```

Comment:
could you please update this to say `is blacklisted` instead of `is blocked`? :)

Reply:
@TroyAlford updated.